index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Portfolio Website</title>
+
+ <!--icon link-->
+ <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css">
+ <script src="https://kit.fontawesome.com/9ea65bb5f9.js" crossorigin="anonymous"></script>
+
+ <!--css link-->
+ <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+  <div class="wrapper">
+    <!--header section-->
+    <header class="header">
+
+      <!--logo-->
+      <a href="#index.html" class="logo">My logo</a>
+
+      <!--navbar-->
+      <nav>
+        <ul class="navlink">
+          <li><a href="#index.html">Portfolio</a></li>
+          <li><a href="#about.html">About</a></li>
+          <li><a href="#contact.html">Contact</a></li>
+        </ul>
+      </nav>
+      <!--end of navbar-->
+
+      <div class="toggle">
+        <i class="uil uil-bars"></i>
+      </div>
+      
+    </header>
+
+    <!--home page-->
+    <section class="home">
+
+      <article class="home-content">
+      <h1 class="home-title">I'm Alexandra <br> Glad to see you!</h1>
+
+      <p class="home-main">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+      <span>LET'S CONNECT</span>
+
+      <ul class="social_links">
+        <li><a href=""><i class="uil uil-twitter"></i></a></li>
+        <li><a href=""><i class="uil uil-linkedin-alt"></i></a></li>
+        <li><a href=""><i class="uil uil-medium-m"></i></a></li>
+        <li><a href=""><i class="uil uil-github-alt"></i></a></li>
+        <li><a href=""><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
```

Comment:
- [ ] I would suggest you please use the same social media icons. You can use it by downloading it one by one from Figma and let's use the same font and same color for the `let's connect` subtitle.
### Your design
![his](https://user-images.githubusercontent.com/81727988/191944043-9cf0cc86-4ad8-4986-ba34-189dbea1cdf5.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191944086-5ab759da-dc38-422e-9e2f-39978fc7b26e.png)